packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-settings/pipeline-menus.tsx

```diff
@@ -26,6 +26,7 @@ function PipelineActionMenu<T extends string>({
   menuItems,
   ['data-testid']: dataTestId,
 }: PipelineActionMenuProp<T>) {
+  const menuTriggerRef = useRef<HTMLButtonElement | null>(null);
```

Comment:
Fixes a bug that cause the menu to miscalculate its height

Reply:
Do we want to have this as a comment in the code? Doesn't look like it is something about height so could be useful for history.